docs/en/Migration-Guides/OpenIddict-Step-by-Step.md

```diff
@@ -19,62 +19,68 @@ Use the `abp update` command to update your existing application. See [Upgrading
 ### Domain.Shared Layer
 
 - In **MyApplication.Domain.Shared.csproj** replace **project reference**:
-  ```csharp
```

Comment:
ℹ️ In these cases, the indentation messed with things a little it. As far as I could tell, removing the indent didn't harm anything.